### PR TITLE
add tests for lifecycle loader

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by adamruzicka — APPROVED** [review]
LGTM, thank you for stepping in and taking it yourself

**Comment by adamruzicka** [comment]
Thank you @evgeni !

File: test/test_plugin_helper.rb [hunk 1]
```diff
@@ -15,3 +15,9 @@ def disable_webpack
     Webpack::Rails::Manifest.stubs(:asset_paths).returns([])
   end
 end
+
+def theme_fixture_file(filename)
+  File.join(
+    ForemanThemeSatellite::Engine.root, 'test', 'fixtures', filename
+  )
```
Comment (ekohl) [inline test/test_plugin_helper.rb:22]: FYI: `ForemanThemeSatellite::Engine.root` is a [Pathname](https://docs.ruby-lang.org/en/3.2/Pathname.html) so you can do
```suggestion
  ForemanThemeSatellite::Engine.root.join('test', 'fixtures', filename)
```

Comment (evgeni) [reply test/test_plugin_helper.rb:22]: https://github.com/RedHatSatellite/foreman_theme_satellite/pull/18

**Pull request merged**